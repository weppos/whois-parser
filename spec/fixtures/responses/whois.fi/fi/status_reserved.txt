
Domain not available

Copyright (c) Finnish Transport and Communications Agency Traficom

