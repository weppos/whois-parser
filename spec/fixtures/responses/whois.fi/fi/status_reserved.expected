#disclaimer
  %s == "Copyright (c) Finnish Transport and Communications Agency Traficom"


#domain
  %s == nil

#domain_id
  %s %ERROR{AttributeNotSupported}


#status
  %s == :reserved

#available?
  %s == false

#reserved?
  %s == true


#created_on
  %s == nil

#updated_on
  %s == nil

#expires_on
  %s == nil


#registrar
  %s == nil

#registrant_contacts
  %s %ERROR{AttributeNotSupported}

#admin_contacts
  %s %CLASS{array}
  %s == []

#technical_contacts
  %s %CLASS{array}
  %s == []


#nameservers
  %s %CLASS{array}
  %s == []
